src/WebJobs.Script/Description/CSharp/CSharpFunctionInvoker.cs

```diff
@@ -97,7 +101,7 @@ protected override void OnScriptFileChanged(object sender, FileSystemEventArgs e
         private void ReloadScript()
         {
             // Reset cached function
-            _function = null;
+            ResetFunctionValue();
             TraceWriter.Verbose(string.Format(CultureInfo.InvariantCulture, "Script for function '{0}' changed. Reloading.", Metadata.Name));
 
             TraceWriter.Verbose("Compiling function script.");
```

Comment:
I'm not sure the elapsed time we're outputting is something the user needs/wants to see. Early on it was useful for us to prove to ourselves that it was fast, but do we really need it in the logs each time?